static/canvas.js

```diff
@@ -151,38 +151,25 @@ class Canvas {
             myState.valid = false;
         }, true);
 
-        /* Shifts through selection of shapes using spacebar. */
```

Comment:
:+1: 
Or, shift-clicking the overlapped region cycles through all the objects under the mouse, is how I've seen some other programs do this